tools/buildifier.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# rules_go still uses the deprecated APIs for args.add, so we explicitly set
```

Comment:
What does `rules_go` have to do with anything?

Reply:
Done (clarify why rules_go is relevant)